lasagne/layers/pool.py

```diff
@@ -145,11 +151,11 @@ def __init__(self, incoming, ds, st=None,
         ----------
         incoming : a :class:`Layer` instance or tuple
             The layer feeding into this layer, or the expected input shape.
-        ds : integer or iterable
+        pool_size : integer or iterable
             The length of the pooling region in each dimension
```

Comment:
That's quite elegant, actually. The text fits both the integer and iterable case.